New landing page

KyoshiNoda opened the pull request:
## Description
- Revamped Landing page as per figma design.
- I used placeholder images for many of the upcoming features we have planned.

## What type of PR is this? (check all applicable)

- [] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] ✨ Enhancement
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
#265 
## Mobile & Desktop Screenshots/Recordings


https://github.com/KyoshiNoda/ScheduleFinder/assets/62672803/f71598cd-51cc-412f-b2c8-f2bf2383e9f1






## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## Additional information

<!-- Is there any additional information that would help understand the changes made by this PR? -->

CDDR1 commented:
It looks great! I really looks how it came about. I have some picky suggestions: 

- I like the CTA button in mobile under the "Create schedules, compare with friends..." subheading. I think it makes sense to keep it in large screens too. 

- I feel like the hero section looks a little bit crammed so adding vertical spacing in between could help.

- The images under the features section look too wide and a bit short in height. I would also suggest adding more horizontal negative space so that there is more space between the borders of the page and the features section. This should also help to make the images taller.

- The space between the hero section and the features section is a bit bigger than the space between the features section and the testimonials. It's no big deal but I suggest making it the same to keep it consistent. 


Other than that it looks great! I'm by no means a product designer but these suggestions are made based on some of the UX/UI tips I've learned :)

KyoshiNoda commented:
@CDDR1
- Addressed all changes, pull down the branch to check it out!
- Additionally, I also added a sticky navbar so when you scroll, you can see the navbar the whole time.
